python/chronos/src/bigdl/chronos/forecaster/autoformer_forecaster.py

```diff
@@ -148,13 +158,134 @@ def __init__(self,
         self.onnx_available = False
         self.quantize_available = False
         self.use_amp = False
+        self.use_hpo = True
 
-        # seed setting
-        from pytorch_lightning import seed_everything
-        seed_everything(seed=self.seed)
+        has_space = _config_has_search_space(
+            config={**self.model_config, **self.optim_config,
+                    **self.loss_config, **self.data_config})
+
+        if not has_space:
+            if self.use_hpo:
+                warnings.warn("HPO is enabled but no spaces is specified, so disable HPO.")
```

Comment:
Small comment(you may ignore this to next PR): This logic will cause those users who just want to use fixed parameters in Autoformer always see this warning.

Reply:
may be solved in next PR : )